recipes/fluidsynth/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir build
+cd build
+cmake .. -Denable-framework=OFF -DLIB_SUFFIX="" -Denable-libsndfile=ON -DCMAKE_INSTALL_PREFIX=$PREFIX
```

Comment:
I think you will need to also add the option ``-DCMAKE_PREFIX_PATH:PATH=$PREFIX``